nix/bazel.nix

```diff
@@ -23,6 +22,13 @@ rec {
     zip
     ;
 
+  # We need to have a file in GOPATH that we can use as
+  # root_file in go_wrap_sdk.
+  go = pkgs.go.overrideAttrs (oldAttrs: {
+    doCheck = false;
+    postFixup = ''touch $out/share/go/ROOT'';
```

Comment:
Did they drop the `README.md` or why is this necessary?

Reply:
Yep, the README.md is gone.